libs/kafka/topichandler.py

```diff
@@ -15,10 +15,11 @@ def create_topic_if_not_exists(kafkaserver, topicname):
     '''
     try:
         consumer = KafkaConsumer(bootstrap_servers=kafkaserver, group_id='libary')
-        if (topics := consumer.topics()) is not None and not topicname in topics:
-            admin_client = KafkaAdminClient(bootstrap_servers=kafkaserver, client_id='lib', api_version=(2, 7, 0))
-            topic_list = [NewTopic(name=topicname, num_partitions=1, replication_factor=1)]
-            admin_client.create_topics(new_topics=topic_list, validate_only=False)
-            print("[+] Created a new topic. Name: {}".format(topicname))
+        if topicname is not None:
+            if (topics := consumer.topics()) is not None and not topicname in topics:
+                admin_client = KafkaAdminClient(bootstrap_servers=kafkaserver, client_id='lib', api_version=(2, 7, 0))
+                topic_list = [NewTopic(name=topicname, num_partitions=1, replication_factor=1)]
+                admin_client.create_topics(new_topics=topic_list, validate_only=False)
+                print("[+] Created a new topic. Name: {}".format(topicname))
     except Exception as error:
         print(error)
```

Comment:
Logging missing?

Reply:
Yes. I was not sure, if it is missing for a particular reason.
If not, I could add it.

Reply:
Would be great. I missed it.

Reply:
Fixed.

Reply:
Tests do fail.

Reply:
I think the logging caused the circular dependency. I removed it for now. The tests should be working again.

Reply:
Merge?

Reply:
From my side ok.